### PR TITLE
Pv2 2011 metric period end stateless service

### PR DESCRIPTION
**Pull request by keenesolutions**

### CODE REVIEW

File: src/SFA.DAS.Payments.Monitoring.Metrics.PeriodEndService/PackageRoot/Config/Settings.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+<Settings xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://schemas.microsoft.com/2011/01/fabric">
+  <!-- Add your custom configuration sections and parameters here -->
+  <!--
+  <Section Name="MyConfigSection">
+    <Parameter Name="MyParameter" Value="Value1" />
+  </Section>
+  -->
+  <Section Name="ConnectionStringsSection">
+    <Parameter Name="StorageConnectionString" Value="UseDevelopmentStorage=true" />
+    <Parameter Name="LoggingConnectionString" Value="Server=.;Database=AppLog;User Id=SFActor; Password=SFActor;" />
```
Comment (hardik-desai-kcl) [inline src/SFA.DAS.Payments.Monitoring.Metrics.PeriodEndService/PackageRoot/Config/Settings.xml:11]: I dont think this is in use why add it for new services

File: src/SFA.DAS.Payments.Monitoring.ServiceFabric/ApplicationPackageRoot/ApplicationManifest.xml [hunk 2]
```diff
@@ -36,6 +39,41 @@
...
+    <ServiceManifestRef ServiceManifestName="SFA.DAS.Payments.Monitoring.Metrics.PeriodEndServicePkg" ServiceManifestVersion="1.0.0" />
+    <ConfigOverrides>
+      <ConfigOverride Name="Config">
+        <Settings>
+          <Section Name="ConnectionStringsSection">
+            <Parameter Name="DcEarningsConnectionString" Value="[DcEarningsConnectionString]" />
+            <Parameter Name="StorageConnectionString" Value="[StorageConnectionString]" />
+            <Parameter Name="ServiceBusConnectionString" Value="[ServiceBusConnectionString]" />
+            <Parameter Name="MonitoringServiceBusConnectionString" Value="[MonitoringServiceBusConnectionString]" />
+            <Parameter Name="LoggingConnectionString" Value="[LoggingConnectionString]" />
```
Comment (hardik-desai-kcl) [inline src/SFA.DAS.Payments.Monitoring.ServiceFabric/ApplicationPackageRoot/ApplicationManifest.xml:52]: I dont think this is in use why add it for new services

File: src/SFA.DAS.Payments.Monitoring.Metrics.PeriodEndService/PackageRoot/Config/Settings.xml (hunk 1 above)
Comment (keenesolutions) [reply src/SFA.DAS.Payments.Monitoring.Metrics.PeriodEndService/PackageRoot/Config/Settings.xml:11]: Discussed offline and agreed to leave in place.

File: src/SFA.DAS.Payments.Monitoring.ServiceFabric/ApplicationPackageRoot/ApplicationManifest.xml (hunk 2 above)
Comment (keenesolutions) [reply src/SFA.DAS.Payments.Monitoring.ServiceFabric/ApplicationPackageRoot/ApplicationManifest.xml:52]: Discussed offline and agreed to leave in place.

**Pull request merged**